app/server.go

```diff
@@ -186,6 +189,10 @@ func StartServer() {
 		}()
 	}
 
+	if *utils.Cfg.PluginSettings.Enable {
+		StartupPlugins("plugins", "webapp/dist")
```

Comment:
How will this work when we decouple the webapp from the server?

Reply:
Basically, it won't. We'll either need to come up with a solution to deal with that or provide instructions for manual installation. I'm pretty sure we're not changing how the webapp is deployed from the server anytime soon anyways and the current way will always be the default. We will just allow the option to host the webapp somewhere else if you choose